server/coordinator.go

```diff
@@ -141,14 +141,17 @@ func (c *coordinator) patrolRegions() {
 				continue
 			}
 
-			key = region.GetEndKey()
-
-			if c.checkRegion(region) {
+			if !c.checkRegion(region) {
```

Comment:
When it returns false, it does not mean the created operator is not successfully added. It's also possible that no operator is needed.

Reply:
Why do we break here? What about the other scanned regions?